From e773dc3c5ce448250e5e75d418aa91f456264c20 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 02 Aug 2022 18:54:21 +0300
Subject: [PATCH] [SW-98239] erom: recovery: remove preboot_dev_enable_caps() call

preboot_dev_enable_caps() function is not imlpemented for Gaudi2.
Remove the callback.

Change-Id: I50f39ef22beaeab72842ca75d8924dca5a0718cc
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 57dbc01..36e995b 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -116,7 +116,6 @@
 	preboot_version();
 
 	preboot_security_sp_sram_init();
-	preboot_dev_enable_caps();
 
 	preboot_clear_wd_gpio();
 
